cmd/web/setup.go

```diff
@@ -2,64 +2,40 @@ package main
 
 import (
 	"log"
-	"os"
 
 	"github.com/joho/godotenv"
 	"github.com/seemsod1/api-project/internal/config"
 	"github.com/seemsod1/api-project/internal/driver"
 	"github.com/seemsod1/api-project/internal/handlers"
-	mailSender "github.com/seemsod1/api-project/internal/mail-sender"
-	"github.com/seemsod1/api-project/internal/models"
+	"github.com/seemsod1/api-project/internal/notifier"
 )
 
 // setup sets up the application
-func setup(app *config.AppConfig) error {
-	env := loadEnv()
-
-	app.Env = env
+func setup(_ *config.AppConfig) error {
```

Comment:
Consider removing this parameter if it is not being used

Reply:
If I remove it here, there will be an error in the main.go, because there I call NewAppConfig, which I don't use, there will be two ways, make _ as a variable, or remove, If remove this line that will mean that I need to remove the AppConfig struct in general. That won't hurt the project, but in the future would be config vars, and I will write the same code again.